tests/test_datasets.py

```diff
@@ -9,6 +9,7 @@
 @dataclass
 class DatasetConfiguration:
     load_function: callable
+    load_kwargs: dict
```

Comment:
Totally a general question, but is there a need to further specify this as `dict[str, Any]`?  `dict` alone leaves it general, but I wasn't sure about best practices.

Reply:
I got lazy with that type annotation, but you're right that keys should always be strings here. Probably good practice to be as specific as possible - fixed!